src/quantum/subbenchmarks.jsx

```diff
@@ -38,6 +41,26 @@ const Graph = ({ data, name, url }) => (
         />
       </a>
     </div>
+    <ChartJsWrapper
+      title={name}
+      type='line'
+      height={80}
+      options={generateOptions()}
+      data={{
```

Comment:
Use `getPerfherderData` and you can pass the `data` it generates.

Reply:
looking at getPerfherderData: it seems to have changed its argument structure a bit since last time.  But it still requires I know the frameworkID, option, and project.  The subbenchmark code only knows suite and platform.